docs/source/tutorials/a_deeper_look.rst

```diff
@@ -50,7 +50,7 @@ the video was generated at 480 resolution at 15 frames per second from the
 ``scene.py`` file.  Therefore, the output can be found inside
 ``media/videos/scene/480p15``.  The additional folders
 ``media/videos/scene/480p15/partial_movie_files`` as well as ``media/text`` and
-``media/Tex`` contain files that are used by manim internally.
+``media/Tex``contains files that are used by manim internally.
```

Comment:
Multiple folders are being talked about, the original is correct.

Reply:
✅